packages/instantsearch-react/docs/package.json

```diff
@@ -6,6 +6,7 @@
   },
   "devDependencies": {
     "babel-cli": "^6.11.4",
+    "babel-plugin-transform-object-rest-spread": "^6.8.0",
```

Comment:
In the end, this isn't needed as it's already present one level above (unless we repeat all babel deps here too).